src/pl/plisql/src/pl_gram.y

```diff
@@ -2236,11 +2316,6 @@ stmt_execsql	: K_IMPORT
 												make_execsql_stmt(T_WORD, @1);
 							new->expr = execstmt->sqlstmt;
```

Comment:
In stmt_execsql rule, can't we just call read_sql_stmt instead of make_execsql_stmt? since we have already determined that its a procedure call...

Reply:
Hi @rasifr , thank you. I've tried this way, but read_sql_stmt doesn't seem to fit here.

Reply:
Hi @rasifr , thank you. I've tried this way, but read_sql_stmt doesn't seem to fit here.